package.json

```diff
@@ -8,32 +8,15 @@
     "url": "git+https://github.com/hashgraph/hedera-sourcify.git"
   },
   "scripts": {
-    "prepublishOnly": "npm install && npm run build",
-    "build": "tsc",
-    "build:clean": "rm -rf node_modules/ packages/bytecode-utils/node_modules/ packages/contract-call-decoder/node_modules/ packages/lib-sourcify/node_modules/ && npm install && npx lerna bootstrap && npx lerna run build",
-    "build:lerna": "lerna run build",
-    "test": "TESTING=true DOTENV_CONFIG_PATH=./environments/.env c8 --reporter=none mocha -r dotenv/config --exit",
-    "test:server": "TESTING=true DOTENV_CONFIG_PATH=./environments/.env mocha -r dotenv/config test/server.js --exit",
-    "test:hedera": "mocha test/test-server.js --exit",
-    "test:chains": "TESTING=true mocha test/chains/chain-tests.js --reporter mochawesome --reporter-options reportDir=chain-tests-report,reportFilename=report --exit",
-    "lint": "eslint . --ext .ts",
-    "lint-tests": "eslint ./test --ext .js",
-    "lint-fix": "eslint . --fix --ext .ts",
-    "coverage": "node ./node_modules/c8/bin/c8.js --reporter=lcov --reporter=text-summary ./node_modules/tape/bin/tape ./test/index.js",
-    "coveralls": "npm run coverage && node ./node_modules/coveralls/bin/coveralls.js <coverage/lcov.info",
-    "server:build": "npm run clean && npm run build",
-    "server:start": "node ./dist/server/server.js",
+    "test:server": "mocha test/server.test.js --exit",
+    "test:hedera": "mocha test/hedera.test.js --exit",
+    "sourcify:patch": "cd sourcify && git apply --verbose ../server.patch",
```

Comment:
Q: should this be server path not sourcify path since the repository is also part of sourcify?